BREEZE.rst

```diff
@@ -233,6 +233,16 @@ Run this command to install Breeze (make sure to use ``-e`` flag):
 Once this is complete, you should have ``breeze`` binary on your PATH and available to run by ``breeze``
 command.
 
+**Docker errors that may come while running breeze**
+
+- If docker not runing in python virtual environment
+- **Solution** : ``sudo chmod 666 /var/run/docker.sock``  
```

Comment:
This is a very bad and extremely insecure solution. You should add your user to docker group. This is the right article describing it in detail https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user

Reply:
oh okay, i will do that.